### ISSUE TITLE
Can't replace aapt2-*-linux.jar

### ISSUE THREAD

**Issue by EverythingVulnerable** [Lzhiyong/termux-ndk#41]
Seems like Google has updated their system. Last week I found the aapt2 jar for 4.1.2 then replaced it with yours but today I'm not able to find aapt2 jar for 4.2.0 AGP. I again tried with 4.1.2 but same, not found. Now what I need to replace?

**Comment by Lzhiyong** [comment]
No problem on my device, I use 4.1.2 to replace 4.2.0, because I have not compiled the 4.2.x version yet
![IMG_20210625_184452](https://user-images.githubusercontent.com/36750502/123414316-51699380-d5e6-11eb-91af-59c3ea1be0c2.jpg)

**Comment by Lzhiyong** [comment]
![IMG_20210625_185344](https://user-images.githubusercontent.com/36750502/123414616-b329fd80-d5e6-11eb-9753-776ef536a535.jpg)

No problem on my device, I use 4.1.2 to replace 4.2.0, because I have not compiled the 4.2.x version yet

**Comment by EverythingVulnerable** [comment]
Thanks a lot for the reply!❣️

Actually, I hadn't built the project when I tried to find the aapt2-*-.jar, so I couldn't find it. But after attempting to build it for the first time and it failed, then I was able to find and replace it! I just totally forgot that the jar needs to be downloaded and placed in the right place. 😅😅

Btw you know, I also use the 4.1.2 in place of 4.2.0 😂 and hopefully it doesn't create any issue. But I request you to build and upload the 4.2.0 or its latest version (4.2.x) so that no chance of issues remains.
